crates/router/src/connector/aci/transformers.rs

```diff
@@ -59,6 +71,162 @@ pub enum PaymentDetails {
+                    bank_account_bic: None,
+                    bank_account_iban: None,
+                    billing_country: None,
+                    merchant_customer_id: None,
+                    merchant_transaction_id: None,
+                    customer_email: None,
+                }))
+            }
+            api_models::payments::BankRedirectData::Sofort { country, .. } => {
+                Self::BankRedirect(Box::new(BankRedirectionPMData {
+                    payment_brand: PaymentBrand::Sofortueberweisung,
+                    bank_account_country: Some(*country),
+                    bank_account_bank_name: None,
+                    bank_account_bic: None,
+                    bank_account_iban: None,
+                    billing_country: None,
+                    merchant_customer_id: None,
+                    merchant_transaction_id: None,
+                    customer_email: None,
+                }))
+            }
+            api_models::payments::BankRedirectData::Przelewy24 {
+                billing_details, ..
+            } => Self::BankRedirect(Box::new(BankRedirectionPMData {
+                payment_brand: PaymentBrand::Przelewy,
+                bank_account_country: None,
+                bank_account_bank_name: None,
+                bank_account_bic: None,
+                bank_account_iban: None,
+                billing_country: None,
+                merchant_customer_id: None,
+                merchant_transaction_id: None,
+                customer_email: billing_details.email.to_owned(),
+            })),
+            api_models::payments::BankRedirectData::Interac { email, country } => {
+                Self::BankRedirect(Box::new(BankRedirectionPMData {
+                    payment_brand: PaymentBrand::InteracOnline,
+                    bank_account_country: Some(*country),
+                    bank_account_bank_name: None,
+                    bank_account_bic: None,
+                    bank_account_iban: None,
+                    billing_country: None,
+                    merchant_customer_id: None,
+                    merchant_transaction_id: None,
+                    customer_email: Some(email.to_owned()),
+                }))
+            }
+            api_models::payments::BankRedirectData::Trustly { country } => {
+                Self::BankRedirect(Box::new(BankRedirectionPMData {
+                    payment_brand: PaymentBrand::Trustly,
+                    bank_account_country: None,
+                    bank_account_bank_name: None,
+                    bank_account_bic: None,
+                    bank_account_iban: None,
+                    billing_country: Some(*country),
+                    merchant_customer_id: Some(Secret::new(item.customer_id.clone().ok_or(
+                        errors::ConnectorError::MissingRequiredField {
+                            field_name: "customer_id",
+                        },
+                    )?)),
```

Comment:
Discussed about this with Arjun, single use and multi use are being handled in core itself rather than connector level so it will remain the same for both single and multi use mandate

Reply:
@Sakilmostak above suggested change missed